runtime/kusama/src/lib.rs

```diff
@@ -1390,6 +1404,15 @@ construct_runtime! {
 		ImOnline: pallet_im_online::{Pallet, Call, Storage, Event<T>, ValidateUnsigned, Config<T>} = 11,
 		AuthorityDiscovery: pallet_authority_discovery::{Pallet, Config<T>} = 12,
 
+		// Old, deprecated governance pallets. Here purely so storage can be migrated.
+		// See https://github.com/paritytech/polkadot/issues/6749
+		Democracy: pallet_democracy exclude_parts {Call} = 13,
```

Comment:
I don't think that we need to add the pallets to construct runtime? Implementing the config traits and then referencing the migrations should be enough?

Reply:
This was discussed in Element some time ago, removing them leads to compilation issues. 

It was suggested instead to leave them in the runtime, but exclude_parts the `Call` to ensure the pallets are inoperable. 

<img width="1002" alt="Screenshot 2023-05-31 at 13 36 18" src="https://github.com/paritytech/polkadot/assets/16665596/673cd015-6e66-488a-9abe-5aba03ba712e">

Reply:
I believe the way to do it properly would be to create a dummy RuntimeEvent that implements the `From` and pass that instead of the actual RuntimeEvent.

Something like
```rust
struct DepRuntimeEvent;
impl From<pallet_democracy::Event<Runtime>> for DepRuntimeEvent {
	fn from(_: pallet_democracy::Event<Runtime>) -> Self {
		Self
	}
}
```

This can be entirely dummy, just enough to make the trait bounds pass I think. 

You would probably need a similar RuntimeOrigin and RuntimeCall.

Reply:
This unfortunately didn't work, there're more errors. Welcome to try it yourself if you like. Otherwise we can pursue the `disable-hooks` approach.

Reply:
> This was discussed in Element some time ago, removing them leads to compilation issues. 
> 
> It was suggested instead to leave them in the runtime, but `exclude_parts` the `Call` to ensure the pallets are inoperable. 
> 
> <img width="1002" alt="Screenshot 2023-05-31 at 13 36 18" src="https://github.com/paritytech/polkadot/assets/16665596/673cd015-6e66-488a-9abe-5aba03ba712e">
> 

You can just implement the requested trait bound yourself.

Reply:
This is not so trivial and led to more compiler issues, but I'll have another go and ask for help if I get stuck again.